### DISCUSSION TITLE
In production, Which one between mix coordinator or 4 coordinator is better?

### DISCUSSION THREAD

**Discussion by anhnch30820** [milvus-io/milvus#36637] [Q&A and General discussion]
I see the default config milvus has been moved to mix coordinator. Currently I am running with the config of 4 coordinator.

**Comment by yhmo** [comment]
In my opinion, one mix coordinator is easier to maintain than 4 coordinators.

**Reply by jubingc** [reply]
How about when things go wrong? Let's say when the data coordinator is down and all other coordinators are still up. So the data ingestion is impacted, but all other functions such as query are still functioning.

I have the mix coordinator deployed with active standby enabled. If I understand correctly, the standby mixcoord only takes over after the active one is down for 60 seconds. Does it mean during these 60 seconds, all functions, including data ingestion and query, are not functioning? 

Do all coordinator roles: data, query, and index run independently using different threads inside the mix coordinator? I try to understand how the one coordinator role impacts the othes in the mix coordinator. Thanks!

**Reply by yhmo** [reply]
Depends on your usage. Data coordinator affects data ingestion, query coordinator affects search/query. If root coordinator stops, the entire service cannot work.
For less maintenance, use one miscoord.
For high-available services, use 4 coordinators.
